### DISCUSSION TITLE
Beat Harvester with Auger wagons

### DISCUSSION THREAD

**Discussion by ServiusTheBearOriginal** [Courseplay/Courseplay_FS22#1643] [Q&A]
Hi, I have noticed that when using the tractor pulled beat harvester it is not lining up to unload with the Auger wagon. It pulls the piper out but stops about a meter in front of it with it using course play. Any ideas?

**Comment by Tensuko** [comment]
Could you explain more on what you try to do?
You talk about self unload ?

**Comment by ServiusTheBearOriginal** [comment]
So when using course play the auto unloading it doesnt stop where the auger is. It stops before it
![20220530202834_1](https://user-images.githubusercontent.com/98311376/171057198-08ade050-1b38-4171-9be2-bb6b5bd182f2.jpg)

**Reply by Tensuko** [reply]
Thats selfunload. Need a logfile with debug for it.

**Reply by Hilbert1982** [reply]
maby it's posible to get a point within the course editor to tel courseplay to drive to there and than unload?

**Reply by Tensuko** [reply]
No
